tests/functional/jobscript/01-share-lib-bin/flow.cylc

```diff
@@ -0,0 +1,33 @@
+#!jinja2
+[scheduler]
+    [[events]]
+        stall timeout = PT0M
+
+[scheduling]
+    initial cycle point = 1266
+    [[graph]]
+        R1 = install_cold => run_mypythonscript & run_myscript
+
+[runtime]
+    [[install_cold]]
+        script = """
+            cat > "${CYLC_WORKFLOW_SHARE_DIR}/lib/python/mypythonscript.py" <<__HERE__
+            def my_function():
+                print("I can speak Esparanto like a native.")
+            __HERE__
+
+            cat > "${CYLC_WORKFLOW_SHARE_DIR}/bin/myscript.sh" <<__HERE__
+                #!/usr/bin/bash
+                echo "A sure cure for seasickness is to sit under a tree."
```

Comment:
Or is it?

![_88932705_20302](https://github.com/cylc/cylc-flow/assets/2362137/bcc663a4-827e-433a-8a7b-c75cdd8a37ec)

Reply:
I feel that one large wave might cure seasickness (and all other illnesses) in a rather permanent way for sailors on _that_.

Reply:
@hjoliver that is an insanely niche reference.

Reply:
https://www.bbc.co.uk/news/blogs-news-from-elsewhere-35891764